src/data/set/intervals/monotone.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 -/
 import data.set.intervals.disjoint
+import data.nat.succ_pred
```

Comment:
Do you need `succ`/`pred` on `nat`, or only generic theory?

Reply:
I probably only need generic theory. I'll have to wait for cache before I can check

Reply:
You can run `leanproject get-cache --rev=master` or `leanproject get-cache --rev=HEAD^`